### PR TITLE
Multiple public ip support

### PR DESCRIPTION
**Pull request by mjmenger**

### CODE REVIEW

File: bigip.tf
Comment (aknot242) [inline bigip.tf:29]: if this 'multiple-public-ips' suffix is removed, will it use master? Should we be using that instead since the intent is to merge this to master after approval?

File: findthehosts.sh
Comment (aknot242) [inline findthehosts.sh]: typo, please remove the '='

**Review by aknot242 — CHANGES_REQUESTED** [review]

File: bigip.tf
Comment (mjmenger) [reply bigip.tf:29]: If we merge that to master we'll be updating the terraform registry module. there are a number of folk that i need to discuss that with before that happens. in the short term we need to pin this way.

File: findthehosts.sh
Comment (mjmenger) [reply findthehosts.sh]: fixed with latest commit

File: bigip.tf
Comment (aknot242) [reply bigip.tf:29]: Ohhhh, sorry. I missed that part. Nothing to see here...move along...

**Review by aknot242 — APPROVED** [review]

**Pull request merged**
